homeassistant/components/lg_netcast/manifest.json

```diff
@@ -1,9 +1,12 @@
 {
   "domain": "lg_netcast",
   "name": "LG Netcast",
-  "codeowners": ["@Drafteed"],
+  "codeowners": ["@Drafteed", "@splinter98"],
+  "config_flow": true,
+  "dependencies": ["network"],
   "documentation": "https://www.home-assistant.io/integrations/lg_netcast",
+  "integration_type": "device",
   "iot_class": "local_polling",
   "loggers": ["pylgnetcast"],
-  "requirements": ["pylgnetcast==0.3.7"]
+  "requirements": ["pylgnetcast==0.3.9"]
```

Comment:
Does the dependency bump need to go hand in hand with adding the config flow? If not, please move the dependency bump to a separate PR.

Reply:
Yes, the dependency bump is here because we've added the `query_device_info` function that is used in the config_flow to get the UUID, model name and friendly name and of the TV.